sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/authentication/ContainerRegistryTokenRequestContextImpl.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry.implementation.authentication;
+
+/**
+ * A token request context associated with a given container registry token.
+ */
+class ContainerRegistryTokenRequestContextImpl {
```

Comment:
Don't need to append `Impl` to class names if they are already in implementation package.